packages/mdc-switch/foundation.ts

```diff
@@ -80,6 +83,14 @@ export class MDCSwitchFoundation extends MDCFoundation<MDCSwitchAdapter> {
       this.adapter_.removeClass(cssClasses.CHECKED);
     }
   }
+
+  private updateAriaChecked_(checked: boolean) {
+    this.adapter_.setNativeControlAttr(
+        strings.ARIA_CHECKED_ATTR,
+        checked ?
```

Comment:
Can we simply use `` `${checked}` `` here and remove the constants?

Reply:
I just thought that in case if `checked` will be some false-like value (if used from JavaScript for example) that `aria-checked` can become invalid. But you are right - using constants looks like overkill too.

Maybe it will be better to use `${!!checked}`? Or just `${checked}` is enough?

Reply:
Good point! `${!!checked}` looks good